### ISSUE TITLE
Typescript definition

### ISSUE THREAD

**Issue by nkint** [academia-de-codigo/react-semantic-toasts#18]
still testing but should be in right way, here is the typescript definition:

```ts
declare module 'react-semantic-toasts' {
  import { ReactNode } from 'react'
  const SemanticToastContainer: () => JSX.Element
  const toast: (
    options: {
      title: string
      description: string
      type: 'info' | 'success' | 'warning' | 'error'
      icon?: any
      time?: number
    },
    onClose?: () => void,
    onClick?: () => void,
  ) => void

  export { SemanticToastContainer, toast }
}
```

**Comment by Evanion** [comment]
Since `semantic-ui-react` is a dependency we can get the icon type from there.
I also added the missing properties for the container.
```
declare module 'react-semantic-toasts' {
  import { ReactNode } from 'react'
  import { SemanticICONS } from 'semantic-ui-react'
  const SemanticToastContainer: (
    props: {
      position?: 'top-right' | 'top-center' | 'top-left' | 'bottom-right' | 'bottom-center' | 'bottom-left'
      animation?: string
    }
  ) => JSX.Element
  const toast: (
    options: {
      title: string
      description: string
      type: 'info' | 'success' | 'warning' | 'error'
      icon?: SemanticICONS
      time?: number
    },
    onClose?: () => void,
    onClick?: () => void,
  ) => void

  export { SemanticToastContainer, toast };
};
```

**Comment by ferrao** [comment]
@nkint @Evanion I have not knowledge of typescript whatsoever, if you guys want to submit a PR feel free to go ahead!

**Comment by StevensonNelli** [comment]
@ferrao @mjsarfatti @nkint I am getting the type error still. Is there any instant solution ?

**Comment by StevensonNelli** [comment]
Yes, got the instant solution.
Followed the [link](https://stackoverflow.com/questions/38971984/how-to-add-custom-typings-in-typescript-2-0) and added custom type in root application.
Thanks.

**Comment by ferrao** [comment]
@mstlaurent-paradigm, is this fixed in either #33 or #31?

**Comment by mstlaurent-paradigm** [comment]
This issue should be fixed following #33.
